bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -17,16 +17,19 @@
 		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="siren-status">
-		<item-type>String</item-type>
-		<label>Status</label>
+	<channel-type id="siren-status-ro">
+		<item-type>Switch</item-type>
+		<label>Siren</label>
 		<description>Status of the siren</description>
-		<state pattern="%s" readOnly="true">
-			<options>
-				<option value="no_sound">Silent</option>
-				<option value="sound">Alarm</option>
-			</options>
-		</state>
+		<category>Soundvolume</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="siren-status-rw">
```

Comment:
I expect an adjustment in the default translation file (new entries).

Reply:
properties file regenerated